gptme/chat.py

```diff
@@ -296,7 +310,17 @@ def step(
 def prompt_user(value=None) -> str:  # pragma: no cover
     print_bell()
     # Flush stdin to clear any buffered input before prompting
-    termios.tcflush(sys.stdin, termios.TCIFLUSH)
+    if termios:
+        termios.tcflush(sys.stdin, termios.TCIFLUSH)
+    elif winpty:
+        if hasattr(winpty, 'flush'):
+            winpty.flush()
```

Comment:
curious as to why this is necessary, given that you will know the exact version installed beforehand

Reply:
msys2 pywinpty didnt have flish and i like using system libraries... will think about removing otherwise